diff --git a/src/contrib/CMakeLists.txt b/src/contrib/CMakeLists.txt
index 7af6804..7252b73 100644
--- a/src/contrib/CMakeLists.txt
+++ b/src/contrib/CMakeLists.txt
@@ -67,7 +67,6 @@ target_link_libraries(lucene++-contrib
     Boost::filesystem
     Boost::iostreams
     Boost::regex
-    Boost::system
     Boost::thread
     ZLIB::ZLIB
     lucene++::lucene++)
diff --git a/src/core/CMakeLists.txt b/src/core/CMakeLists.txt
index 690f382..657de54 100644
--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -57,7 +57,6 @@ target_link_libraries(lucene++
     Boost::filesystem
     Boost::iostreams
     Boost::regex
-    Boost::system
     Boost::thread
     ZLIB::ZLIB
 )
diff --git a/src/demo/deletefiles/CMakeLists.txt b/src/demo/deletefiles/CMakeLists.txt
index b50d5be..4213195 100644
--- a/src/demo/deletefiles/CMakeLists.txt
+++ b/src/demo/deletefiles/CMakeLists.txt
@@ -38,7 +38,6 @@ target_link_libraries(deletefiles
     Boost::filesystem
     Boost::iostreams
     Boost::regex
-    Boost::system
     Boost::thread
     ZLIB::ZLIB
     lucene++::lucene++
diff --git a/src/demo/indexfiles/CMakeLists.txt b/src/demo/indexfiles/CMakeLists.txt
index a1dd582..68ddced 100644
--- a/src/demo/indexfiles/CMakeLists.txt
+++ b/src/demo/indexfiles/CMakeLists.txt
@@ -39,7 +39,6 @@ target_link_libraries(indexfiles
     Boost::filesystem
     Boost::iostreams
     Boost::regex
-    Boost::system
     Boost::thread
     ZLIB::ZLIB
     lucene++::lucene++
diff --git a/src/demo/searchfiles/CMakeLists.txt b/src/demo/searchfiles/CMakeLists.txt
index aaf559e..dda9542 100644
--- a/src/demo/searchfiles/CMakeLists.txt
+++ b/src/demo/searchfiles/CMakeLists.txt
@@ -38,7 +38,6 @@ target_link_libraries(searchfiles
     Boost::filesystem
     Boost::iostreams
     Boost::regex
-    Boost::system
     Boost::thread
     ZLIB::ZLIB
     lucene++::lucene++
diff --git a/src/test/CMakeLists.txt b/src/test/CMakeLists.txt
index f0b9b7e..69393c0 100644
--- a/src/test/CMakeLists.txt
+++ b/src/test/CMakeLists.txt
@@ -59,7 +59,6 @@ target_link_libraries(lucene++-tester
     Boost::filesystem
     Boost::iostreams
     Boost::regex
-    Boost::system
     Boost::thread
     ZLIB::ZLIB
     gtest_main
diff --git a/cmake/dependencies.cmake b/cmake/dependencies.cmake
index 74c3761..26f5f0a 100644
--- a/cmake/dependencies.cmake
+++ b/cmake/dependencies.cmake
@@ -7,7 +7,6 @@ find_package(Boost COMPONENTS
   filesystem
   iostreams
   regex
-  system
   thread
   REQUIRED
 )
@@ -20,7 +19,6 @@ set(lucene_boost_libs
   ${Boost_FILESYSTEM_LIBRARIES}
   ${Boost_IOSTREAMS_LIBRARIES}
   ${Boost_REGEX_LIBRARIES}
-  ${Boost_SYSTEM_LIBRARIES}
   ${Boost_THREAD_LIBRARIES}
 )
 
